runtime/doc/options.txt

```diff
@@ -6273,6 +6273,23 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Insert mode completion.  When zero as much space as available is used.
 	|ins-completion-menu|.
 
+						*'pumordering'* *'po'*
+'pumordering' 'po'	string (default "012")
+	This option specify how the popup menu should look like
+
+	The default value:
+	0: word
+	1: kind
+	2: extra
```

Comment:
same here, default value is no longer 0,1,2

Reply:
yea